static/lang/re-en.json

```diff
@@ -2674,6 +2674,9 @@
             "Oracle": {
                 "Mystery": {
                     "Prompt": "Select a mystery."
+                },
+               "MysteryConduit":{
+                    "Note":"When you cast a spell of 5th rank or lower that has no duration, instead of spending a spell slot, you can advance the effects of your oracular curse in the same manner as when you cast a revelation spell. A spell cast in this way gains the cursebound trait."
```

Comment:
```suggestion
                    "Note": "When you cast a spell of 5th rank or lower that has no duration, instead of spending a spell slot, you can advance the effects of your oracular curse in the same manner as when you cast a revelation spell. A spell cast in this way gains the cursebound trait."
```